<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Plot</class>
 <widget class="QMainWindow" name="Plot">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1429</width>
    <height>904</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TATSSI Time Series Analysis</string>
  </property>
  <widget class="QWidget" name="content_plot">
   <widget class="QComboBox" name="data_vars">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>10</y>
      <width>271</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lblDataVariables">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Data variables</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblModel">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>10</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Decomposition model</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbAnomalies">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>10</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Anomalies</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblYears">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Years in time series</string>
    </property>
   </widget>
   <widget class="QComboBox" name="years">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>40</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="model">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>10</y>
      <width>271</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
